testing/npd6/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Kevin Daudt <kdaudt@alpinelinux.org>
+# Maintainer: Kevin Daudt <kdaudt@alpinelinux.org>
+pkgname=npd6
+pkgver=1.1.0
+pkgrel=0
+pkgdesc="Neighbor proxy daemon for IPv6"
+url="https://www.ipsidixit.net/"
+arch="all"
+license="GPL-3.0+"
+makedepends="linux-headers"
+subpackages="$pkgname-doc $pkgname-openrc::noarch"
+options="!check"
```

Comment:
Need comment to reason why passing the tests. Like:
`options="!check" # No test suite`